### ISSUE TITLE
LazyDataModel#getRowData is called twice

### ISSUE THREAD

**Issue by ohlsen3000** [primefaces/primefaces#290]
Szenario:

Configure a table with a `LazyDataModel` and single selection. If you do select an entry in the Table the method `getRowData(final String rowKey)` is called twice. That is rather bad because for `LazyDataModel`s this method is likely to access the database.

I'm using PF 5.2

**Comment by arunvc** [comment]
Datascroller with a LazyDataModel and the load method from lazy data model is getting called twice.
Will this fix solve data scroller issue?

http://stackoverflow.com/questions/29303228/primefaces-lazy-datascroller-calling-load-twice

**Comment by tandraschko** [comment]
nope, don't think so.

**Comment by arunvc** [comment]
Shall i report the Datascroller load method twice called issue as a new bug.

// BUGFIX: outcommented as this is already done in loadChunk() later on.
/_if(ds.isLazy()) {
    loadLazyData(ds, 0, chunkSize);
}_/

**Comment by tandraschko** [comment]
plz check before if the issue doesn't exist already.

**Comment by cagataycivici** [comment]
This code is common in all components that use LazyDataModel so I don't think a new issue is required.

**Comment by nunofaria11** [comment]
I'm the author of the mentioned question and answer on [SO](http://stackoverflow.com/questions/29303228/primefaces-lazy-datascroller-calling-load-twice) - I don't believe it is a `LazyDataModel` issue, but a `DataScrollerRenderer` issue. 
The suggested correction works for me - the `DataScroller` now inits data only once on render.

**Comment by cagataycivici** [comment]
Ok closing and will look at #390
